Knossos.NET/Views/Windows/ModSettingsView.axaml

```diff
@@ -34,6 +34,17 @@
 						<Label Foreground="White">Available exec type options: </Label>
```

Comment:
So yes, to clarify what I said on discord, changing height on for this view model to 650 (line 750) seems to be much more usable for users with smaller screens.